fe/fe-core/src/main/java/org/apache/doris/common/util/Util.java

```diff
@@ -477,5 +478,14 @@ public static InputStream getInputStreamFromUrl(String urlStr, String encodedAut
         conn.setReadTimeout(readTimeoutMs);
         return conn.getInputStream();
     }
+
+    public static boolean showHiddenColumns() {
+        try {
+            return ConnectContext.get().getSessionVariable().showHiddenColumns();
```

Comment:
ConnectContext.get() may return null, better check it first, and if it is null, return a default value.